ApollosProject/apollos-apps
# chore: Daily Habit Copy Updates

## ✏️ Description
We need to be able to support custom typography for certain aspects of Daily Habits. This PR introduces a couple of copy changes to the default copy and also enables customization for some areas.

1. Changes the default copy of the IDLE Hero Card title
2. Changes the default copy IN PROGRESS Hero Card title
3. Changes the default copy of the Today Gratitude title
4. Changes the default copy of the Today Prayer Card title
5. Enables prop overrides on the Today Gratitude Card
6. Enables prop overrides on the Today Prayer Card
7. Enables prop overrides on the Today Reading Card

## 🔬 To Test
1. Open up the `mobile/theme.json`
2. Update the file with the JSON provided below
3. Run the app and navigate to the Daily tab
4. Adjust different copy for the `daily-habits.TodayGratitudeCard`, `daily-habits.TodayPrayerCard`, and `daily-habits.TodayReadingCard` to see the changes _(note: you might need to rebuild the app)_

```json
{
  "colors": {},
  "typography": {},
  "overrides": {
    "daily-habits.TodayGratitudeCard": {
      "title": "Did this work?",
      "summary": "YES IT DID BABAAYYY!!!!"
    }
  }
}

```

## 📸 Screenshots
![Screenshot_20221216-140533](https://user-images.githubusercontent.com/45076058/208174481-c15919eb-e80a-413f-8f0c-086aed948a3c.png)


## ⚠️ To-do Before Merge
@conrad-vanl needs to approve of this before it can get merged

## 📝 Checklist
